### PR TITLE
SSVC-318 Fixing spelling errors adding comments to Master Data endpoints

### PR DESCRIPTION
**Pull request by binghamtj**
@DudeSolutions/software-engineering @ejcarr @brianhyder

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -72,37 +72,45 @@ Any response that does not provide a ```200 OK``` response code should be treate
     + Attributes (Heartbeat)
 
 # Group Master Data
-The master data collection allows for integration between the Dude Platform and the dudes other system of records.
+The master data collection allows for integration between the Dude Platform and the dude's other system of records.
```
Comment (elizabethmidgett) [inline apiary.apib]: Should this say "The Dude's"?

File: apiary.apib [hunk 2]
```diff
@@ -72,37 +72,45 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
 
-## GetUser [/masterdata/users/{userName}]
+## Get User [/masterdata/users/{userName}]
+The get user is exposed and provided to return a dude platform user from a system of record user name.
```
Comment (elizabethmidgett) [inline apiary.apib]: "Dude Platform"? (I'm not really sure when our names should be proper names?)

Comment (binghamtj) [inline apiary.apib]: Updating to "Dude Platform" talking with brent and this is how to name it.

File: apiary.apib (hunk 1 above)
Comment (binghamtj) [inline apiary.apib]: changed to be Dude Solutions' because I didn't like "the dude's" anyways

**Pull request merged**